### PR TITLE
Fix/lw 7562 typeorm stakepool provider

### PR DESCRIPTION
**Pull request by iadmytro**
# Context

Updated Typeorm stake pool provider to pass e2e tests and added to CI e2e suite

# Importand changes

- excluded `cost` from `nullsInSort` 
- updated table names for `cost` and `name` in queries 
- switched to case-insensitive sort by `name`
- added `pledgeMet` filter
- added wildcard search by `name` and `ticker`

### CODE REVIEW

**Review by iccicci — APPROVED** [review]
Great! Just the suggestion for a minor change to reduce some code repetition

File: .github/workflows/continuous-integration-e2e.yaml [hunk 1]
```diff
@@ -85,3 +85,27 @@ jobs:
         STAKE_POOL_PROVIDER: 'http'
         STAKE_POOL_PROVIDER_PARAMS: '{"baseUrl":"http://localhost:4000/stake-pool"}'
         STAKE_POOL_PROVIDER_URL: 'http://localhost:4000/stake-pool'
+
```
[the remark this answers is not in the document]
Comment (iadmytro) [reply .github/workflows/continuous-integration-e2e.yaml:88]: I tried, got error about missing env vars

[the remark this answers is not in the document]
Comment (rhyslbw) [reply .github/workflows/continuous-integration-e2e.yaml:88]: How about loading env from file?

[the remark this answers is not in the document]
Comment (iadmytro) [reply .github/workflows/continuous-integration-e2e.yaml:87]: @rhyslbw  moved env vars from job to global block in [d21db3](https://github.com/input-output-hk/cardano-js-sdk/pull/826/commits/d21db339ba1790a235cb3632ce1e05cbec06cb8c)

**Review by iccicci — APPROVED** [review]

File: .github/workflows/continuous-integration-e2e.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (iadmytro) [reply .github/workflows/continuous-integration-e2e.yaml:87]: also in this commit redundant KORA_LABS vars were removed
```
  KORA_LABS_HANDLE_PROVIDER: 'kora-labs'
  KORA_LABS_HANDLE_PROVIDER_PARAMS: '{"serverUrl":"http://localhost:4000","policyId":""}'
```

**Review by rhyslbw — APPROVED** [review]
Fantastic work @iadmytro 🚀

**Pull request merged**